examples/jsm/exporters/GLTFExporter.js

```diff
@@ -2499,6 +2514,93 @@ class GLTFMaterialsVolumeExtension {
 
 }
 
+/**
+ * Materials ior Extension
+ *
+ * Specification: https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_ior
+ */
+class GLTFMaterialsIorExtension {
+
+	constructor( writer ) {
+
+		this.writer = writer;
+		this.name = 'KHR_materials_ior';
+
+	}
+
+	writeMaterial( material, materialDef ) {
+
+		if ( ! material.isMeshPhysicalMaterial || material.ior === 1.5 ) return;
+
+		const writer = this.writer;
+		const extensionsUsed = writer.extensionsUsed;
+
+		const extensionDef = {};
+
+		extensionDef.ior = material.ior;
+
+		materialDef.extensions = materialDef.extensions || {};
+		materialDef.extensions[ this.name ] = extensionDef;
+
+		extensionsUsed[ this.name ] = true;
+
+	}
+
+}
+
+/**
+ * Materials specular Extension
+ *
+ * Specification: https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_specular
+ */
+class GLTFMaterialsSpecularExtension {
+
+	constructor( writer ) {
+
+		this.writer = writer;
+		this.name = 'KHR_materials_specular';
+
+	}
+
+	writeMaterial( material, materialDef ) {
+
+		if ( ! material.isMeshPhysicalMaterial || ( material.specularIntensity === 1.0 && 
+      material.specularColor.equals(DEFAULT_SPECULAR_COLOR) && 
```

Comment:
Hmm, strange that the linter didn't notice this... @LeviPesin related to the latest CI changes maybe?

Reply:
CI currently still only lints the core /src files - not the example files.

Reply:
Oh right right

Reply:
> @LeviPesin related to the latest CI changes maybe?

No -- I suggested in https://github.com/mrdoob/three.js/pull/25276 to expand the lint script to also check the linting of other directories but that idea didn't recieve much support.
Thinking about it again... maybe we can at least add a commit hook that will automatically run `npm run lint-fix` when user makes a commit via Git?

Reply:
That'd be a good start I think?

Reply:
> Thinking about it again... maybe we can at least add a commit hook that will automatically run npm run lint-fix when user makes a commit via Git?

We have to be careful about how this is done. I commit individual lines when working through feature additions and every time I've used "auto linters" on commit it stages and commits the entire file including lines I didn't want included.

Reply:
I didn't even know about `npm run lint-fix`; that would probably save me a lot next time.